aws-sns-topic/aws-sns-topic.json

```diff
@@ -147,9 +144,7 @@
         },
         "delete": {
             "permissions": [
-                "sns:DeleteTopic",
-                "sns:ListSubscriptionsByTopic",
-                "sns:Unsubscribe"
```

Comment:
what was Unsubscribe used for originally?

Reply:
No, I think this was added 2 yrs ago in Uluru, the logic at that time was, when delete happens, it will try to unsubscribe all subscriptions, then delete topic. But this would be an issue if this topic has a lot subscriptions, also it is different from native logic, which we just delete. So I removed that logic when migration, just forgot to clear these permission that time.